src/irr/asset/CImageLoaderJPG.cpp

```diff
@@ -255,7 +256,7 @@ asset::IAsset* CImageLoaderJPG::loadAsset(io::IReadFile* _file, const asset::IAs
     asset::CImageData* image = 0;
 	if (useCMYK)
 	{
-		image = new asset::CImageData(NULL,nullOffset,imageSize,0,asset::EF_R8G8B8_UNORM);
+		image = new asset::CImageData(NULL,nullOffset,imageSize,0,asset::EF_R8G8B8_SRGB);
```

Comment:
if gamma=2.2 then its SRGB, if gamma 2.0 then you can also pretend its SRGB, if gamma 1.0 then its UNORM

anything else then throw an error